web/src/components/TaskHeader.tsx

```diff
@@ -14,12 +15,28 @@ import {
 } from "@/lib/utils/helpers";
 import React, { ReactElement, useState } from "react";
 import { CongratulatoryDialog } from "./CongratulatoryDialog";
+import { Tag } from "./njwds-extended/Tag";
 
 interface Props {
   task: Task;
   tooltipText?: string;
 }
 
+const RequiredTag = (): ReactElement => (
```

Comment:
if this is going to be its own independent const, I think it should probably live in its own file

Reply:
I'll remove it from an independent const for now and just put it w the rest of the JSX. Easier to read and no current need for a separate component.